articles/openstack/ostack-how-manage-accounts-portal.md

```diff
@@ -73,52 +71,19 @@ To assign or revoke OpenStack permissions:
 
     ![Edit button for Portal contact](images/ptl-contacts-btn-edit.png)
 
-4. On the *Edit contact* page, select the **Permissions** tab.
-
-    ![Permissions tab for Portal contact](images/ptl-contacts-tab-permissions.png)
-
-5. In the *Permissions for UKCloud for OpenStack* section, you'll see a list of the OpenStack projects within your organisation.
+4. On the *Edit contact* page, select the **OpenStack Permissions** tab.
 
-    > [!TIP]
-    > You may need to scroll down the **Permissions** tab to get to the *UKCloud for OpenStack* section.
+    ![Permissions tab for Portal contact](images/ptl-contacts-tab-openstack-permissions.png)
 
-6. For the project to which you want to grant the user access, select the check box for the region in which the project is located. If the project is available in both OpenStack regions (cor00005 or frn00006), you can grant access in just one region or in both.
-
-    > [!NOTE]
-    > If you grant a user access to a region in which the project is not available, the user will not be able to log in to that OpenStack region.
+5. You'll see a list of the OpenStack projects within your organisation. There's a separate entry for each region a project is in.
 
+6. For the project/region to which you want to grant the user access, select the check box.
```

Comment:
Select the check box for the project and region to which you want to grant the user access.

Reply:
Done.